### ISSUE TITLE
Review greys

### ISSUE THREAD

**Issue by yaili** [ubuntudesign/cloud-vanilla-theme#26]
Actions from the team offsite:

- Juju and MAAS will review their greys to make sure they're correct - @jamiedawsonyoung @spencerbygraves
- Juju's light grey could be #f7f7f7 instead of #f5f5f5, to keep consistency
- Check if Juju still needs the light-mid-grey #d4d4d4
- Check if Juju still needs the dark-grey #9fa7aa

**Comment by jamiedawsonyoung** [comment]
Hey there, so all the greys that we decided on for the client work are here: 
https://sites.google.com/a/canonical.com/apps-and-platform-team/3-platform/toolkit/visual-style

It looks like we're close on some of the colours, might be worth having a quick look and see if we can be 100% consistent?

**Comment by yaili** [comment]
Didn't we do that at the workshop?

**Comment by spencerbygraves** [comment]
For Juju, we no longer use #9fa7aa. We still use #d4d4d4, along with #cdcdcd, #888 and #333

**Comment by jamiedawsonyoung** [comment]
Yep! Sorry we did (I forgot) :-)

**Comment by jamiedawsonyoung** [comment]
Just a wee FYI… Andrea filed a bug that saying our text colours (greys) didn't have enough contrast to pass the AAA accessibility standard: https://bugs.launchpad.net/ubuntu-ux/+bug/1579704

It might be that we need to change some of the greys, I've asked Rae to look at the two cases he refers to in the bug. Off the top of my head I think it's fine but worth mentioning.

**Comment by anthonydillon** [comment]
Just checked the MAAS table header styles as its the first grey text I
stumbled on. It failed AAA and AA. Might be something to keep in mind.

**Comment by spencerbygraves** [comment]
Thanks, that's good to know.

I've had a quick check using that tool and it says that #333333 passes when used on our #f7f7f7 background, but #888888 fails. Also, #888888 fails on white.

So, it looks like we should be using #333333 as our text colour, whether on grey or white. Perhaps #888888 is ok for much less important text, what do we think?

**Comment by anthonydillon** [comment]
I think it does matter. We should be running this tool over each of our
pages/views and make sure there are no errors.

http://squizlabs.github.io/HTML_CodeSniffer/

**Comment by spencerbygraves** [comment]
Yep, it definitely matters. I was just wondering if there's still a case for using #888 as a text colour, which we occasionally do.

**Comment by yaili** [comment]
There are some rules we follow on grey text: http://design.canonical.com/2012/01/some-guidelines-for-warm-grey-text-on-the-web/ (note to self: these should be placed in the style guide).

#888888 on white passes AA for larger font size, which is partially what we decided to do in 2012.

I think lately, on ubuntu.com, we've been using the smaller font size instead of grey text for things like footnotes, which is more readable, better contrast. Maybe we should consider phasing it out.

**Comment by jamiedawsonyoung** [comment]
Do you mean phasing out the 888? I think that might be tough to do across the board. I think we should look at where it gets used. I've also found great tool: http://contrast-finder.tanaguru.com/ It does the checking but it also makes suggestions for colours that are close to the one you put in as well as some suggestions that might be better contrast ratio (and close to the original)

For instance #888 might fail on #FFF but #797676 passes AAA
<img width="1189" alt="screenshot 2016-05-11 11 50 12" src="https://cloud.githubusercontent.com/assets/6909617/15192581/c13223da-176e-11e6-925f-aba02463aa3f.png">

#888 on #f7f7f7 is harder. It passes AA (3.1) so is okay for larger text (what is considered large?) Maybe this is where we create a usage rule?

**Comment by anthonydillon** [comment]
I think font weight matters here too. As we use the light font I think we
need to be more strict with our contrast.

**Comment by anthonydillon** [comment]
I tried replacing #888 on Juju to #797676 but it is very close in color to
the text color #333. Not sure it have enough difference to show is sub
content.

**Comment by spencerbygraves** [comment]
#797676 isn't a colour we use for Juju, I'd rather not introduce another one, especially if it's already close to #333333.

I don't think we use #888888 above 16px for any text, so that's where the issue lies.

**Comment by spencerbygraves** [comment]
Sorry, I thought that new value was something from client, I see it's suggested by that tool.

**Comment by jamiedawsonyoung** [comment]
So if 797676 is too close to 333333 then maybe we need to darken the darkest text colour to maintain the hierarchy of primary and secondary. We had to do this in client for that very reason, our main text colour (after a lot of different options) became 111111.

The font weight thing is interesting too, I just read a bunch of use testing results from the client work and almost all of them have that the font is too light, delicate, thin etc. To be easy to scan quickly.

All these suggestions are (maybe technically) relatively easy to implement but they're going to require some work and time to communicate to the designers and Marcus. So would be great to try some combinations in various products / websites to show alongside the explanation of why we need to make the changes. It's not going to be a quick change this I fear... (But it is important and worth doing!)
